code/edgeconsent/build.gradle

```diff
@@ -160,7 +160,7 @@ publishing {
             pom {
                 name = mavenRepoName
                 description = mavenRepoDescription
-                url = 'https://aep-sdks.gitbook.io'
+                url = 'https://developer.adobe.com/client-sdks'
                 licenses {
                     license {
                         name = 'Adobe Proprietary'
```

Comment:
L177 
Let's update the urls in  scm { 
connection
developerConnection 
url
}

refer: https://github.com/adobe/aepsdk-edge-android/blob/dev-v2.0.0/code/edge/build.gradle#L142